routemaster/config/visualisation.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<style>
+body {
+  font: 14px helvetica neue, helvetica, arial, sans-serif;
+  margin: 0;
+}
+#cy {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  left: 0;
+  top: 3.5em;
+}
+#controls {
+  background-color: #f0f0f0;
+  margin: 0;
+}
+#controls div {
+  display: inline-block;
+  padding: 0.5em;
+  margin-right: 0.2em;
+}
+#spacing-factor-input-container {
+  top: 0.5em;
+  position: relative;
+}
+.muted {
+  color: #666666;
+}
+.action {
+  background-color: orange;
+}
+.gate {
+  background-color: lightBlue;
+}
+</style>
+<title>Visualisation of state machine {{ state_machine_name }}</title>
+</head>
+<body>
+  <script type="text/javascript">
```

Comment:
Can we make this `application/json` and remove the var? Adding an ID means we can read in the node from the JS.

Reply:
I originally had that, but I'm not sure what the benefit is? The browser still needs to parse the content as JSON, so I'm not sure what we're saving by serving it up as JSON and then parsing it from JavaScript?

Reply:
JSON is not a strict subset of JS, so `to_json` doesn't _technically_ work in the general case (although I realise we're unlikely to hit the issue). I also prefer this just because it's stricter in general, it means we don't template our Javascript at all, and will hopefully be less likely to introduce templating JS in the future.

Reply:
d5c676dfe003529c513156ac4fea5c7ac79776c7